src/dso_api/dynamic_api/temporal.py

```diff
@@ -166,13 +224,23 @@ def url_parameters(self) -> dict[str, str]:
             return {}
 
 
+def get_request_date(request: Request) -> datetime:
+    """Find the temporal query date, associated with the request object."""
+    try:
+        # Make sure all queries use the exact same now, not seconds/milliseconds apart.
```

Comment:
This is mainly for SQL, right? Can't we use the Postgres [`NOW`](https://www.postgresql.org/docs/14/functions-datetime.html#FUNCTIONS-DATETIME-CURRENT) function instead? That returns the starting time of the transaction.

Reply:
Nope. All the queries for the request should use the same date. Otherwise, there is a 10-100ms difference in the requested date.

Reply:
Do the queries not run within a transaction?

Reply:
Interesting about `NOW`, didn't know that. But that will be tricky, and I don't think there is a transaction at all.